requirements/requirements.txt

```diff
@@ -1,4 +1,4 @@
-ingenialink @ git+https://github.com/ingeniamc/ingenialink-python@develop
+ingenialink @ git+https://github.com/ingeniamc/ingenialink-python@test-INGK-703
```

Comment:
Revert this change

Reply:
Yes, I'll remove it afterward finishing a test.

Reply:
Done!